_posts/blog/2018-11-07/2018-11-07-big-change-hard-work-and-strong-growth-openbazaar-2-turns-1.markdown

```diff
@@ -0,0 +1,53 @@
+
+#### Wonder What's Next?
+
+In 2018 the breadth of crypto projects has increased exponentially through ERC-20 token issuances, completely new blockchains, layer two networks and more. We understand that there is a growing number of people interested in using all kinds of cryptocurrencies—not just Bitcoin—for e-commerce, coin trading, rewarding others for content creation and other use cases. OpenBazaar was always designed as a project for _the world_, not just for Bitcoin, and we’re hard at work re-engineering it to support new coins and communities. 
+
+Our next release will center on the [multiwallet](https://openbazaar.org/blog/Freedom-To-Trade-Means-Freedom-To-Choose-Currencies-The-OpenBazaar-Multiwallet/) which will allow users to send and receive payments from one wallet instance that can hold these five coins: Bitcoin, Bitcoin Cash, Zcash, Ethereum and Litecoin. We hope to introduce this new capability and others very soon.
+
+![OpenBazaar Multiwallet design](openbazaar-multiwallet-design.png "OpenBazaar Multiwallet design for Bitcoin, Bitcoin Cash, Zcash, Ethereum and Litecoin")
+> Want to see more? Click [here](https://projects.invisionapp.com/share/5XMZC8DT8HM#/screens/308962050_Openbazaar-2-0-Wallet) for an interactive demo!
+
+OB1 is also bringing OpenBazaar to mobile with **our own iOS and Android apps** and we’re pioneering the technology to make a truly decentralized web version of OpenBazaar so that anyone can run it in their own standard browser. It’s an extremely bright future and every day thousands of new people are learning about the power of Bitcoin, Ethereum and other crypto projects. 
+
+There are even more exciting developments on the horizon beyond these ambitious releases that I can’t wait to share with you. Please sign up for our [newsletter](https://openbazaar.org/newsletter/) to learn about them as they are announced.
+
+#### OpenBazaar 2 Turns 1
+
+This month we are celebrating one year since the release of OpenBazaar 2.0 with promotions and events all month long. Connect with us on [Reddit](https://reddit.com/r/openbazaar), [Facebook](https://facebook.com/openbazaarproject) and [Slack](https://openbazaar.org/slack) to learn more and follow along on [Twitter](https://twitter.com/openbazaar) with the hashtag [#OB2Turns1](https://twitter.com/search?src=typd&q=%23OB2Turns1).
+
+#### In Conclusion
+
+Every day we use technologies that make our lives easier, but at the expense of our **privacy** or **security**. We’re working on this freedom-focused project to give individuals another choice for how they buy, sell and share their information online. A choice that respects these values we share.
+
+> **This movement is about taking control of the future we want to see for ourselves and our neighbors all around the world.**
+
+We are excited to keep working with you on this project and to help it grow together. We can only do it with your support so please spread the good word and help us make OpenBazaar even better.
```

Comment:
recommendation:"spread the word" vs. "spread the good word", the first is the more typical form of the phrase, and not associated with religion.